tools/ci/tc/decision.py

```diff
@@ -101,17 +100,30 @@ def get_extra_jobs(event):
     return jobs
 
 
-def filter_jobs(jobs, event):
-
-    # Exclude browser stability tests for exports from that specific browser.
+def filter_excluded_users(tasks, event):
+    # Some users' pull requests are excluded from tasks,
+    # such as pull requests from automated exports.
     try:
-        if event["pull_request"]["user"]["login"] == "chromium-wpt-export-bot":
-            jobs.discard("wpt-chrome-dev-stability")
-        if event["pull_request"]["user"]["login"] == "moz-wptsync-bot":
-            jobs.discard("wpt-firefox-nightly-stability")
+        submitter = event["pull_request"]["user"]["login"]
     except KeyError:
-        # Just continue if the username cannot be pulled from the event.
+        # Just ignore excluded users if the
+        # username cannot be pulled from the event.
         logger.debug("Unable to read username from event. Continuing.")
+        return
+
+    excluded_tasks = []
+    # A separate list of items for tasks is needed to iterate over
+    # because removing an item during iteration will raise an error.
+    for name, task in list(tasks.items()):
+        for excluded_user in task.get("exclude-users", []):
```

Comment:
I wonder why we don't just do `if submitter in task.get("exclude-users", [])`

Reply:
Ah, you're right. More concise and efficient, as the check terminates once a match is found. Changed!